pkg/apis/cloud/v1beta1/backend.go

```diff
@@ -38,6 +41,18 @@ type Backend struct {
 
 	// VolumeClaim is an optional persistent volume claim to use for persistence
 	VolumeClaim *corev1.PersistentVolumeClaim `json:"volumeClaim,omitempty"`
+
+	// LivenessProbe backend livenessProbe configuration
+	LivenessProbe *corev1.Probe `json:"livenessProbe,omitempty"`
+
+	// ReadinessProbe backend readinessProbe configuration
+	ReadinessProbe *corev1.Probe `json:"readinessProbe,omitempty"`
+
+	// Args proxy container args
+	Args []string `json:"args,omitempty"`
+
+	// Metadata backend metadata
+	Metadata metav1.ObjectMeta `json:"metadata,omitempty"`
```

Comment:
The idea is to use metada information such as names, labels to reconcile to distinguish backends in different controllers